parts/k8s/containeraddons/kubernetesmasteraddons-azure-npm-daemonset.yaml

```diff
@@ -79,6 +79,11 @@ spec:
           image: {{ContainerImage "azure-npm-daemonset"}}
           securityContext:
             privileged: true
+          resources:
+            requests:
+              memory: 20Mi
+            limits:
+              memory: 2Gi

```

Comment:
And here replace the static `2Gi` with `{{ContainerMemLimits "azure-npm-daemonset"}}`

Reply:
fixed